### PR TITLE
GIX-1047: New transaction modal top up neurons

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Use the new flow to make transctions also when topping up neurons.

# Changes

* Use "TransactionModal" in the "IncreaseStakeModal".
* Remove NewTransactionModal and files under it.
* New neuron service topUpNeuron.
* New neuron util validTopUpAmount.

# Tests

* Change tests in IncreaseStakeModal.
* Test for new neuron service.
* Test for new neuron util.
* Remove tests of removed files.

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UX:

https://user-images.githubusercontent.com/4550653/194828245-4f07d147-a58e-4e4f-b4d7-bca67b5a9391.mov

**Comment by lmuntaner** [comment]
@peterpeterparker @mstrasinskis please review

File: frontend/src/lib/modals/accounts/IcpTransactionModal.svelte [hunk 1]
```diff
@@ -31,16 +30,10 @@
...
-      amount,
-      token: ICPToken,
-    });
-
     const { success } = await transferICP({
-      selectedAccount: sourceAccount,
+      sourceAccount,
       destinationAddress,
-      amount: tokenAmount,
+      amount: amount,
```
Comment (peterpeterparker) [inline frontend/src/lib/modals/accounts/IcpTransactionModal.svelte:36]: nth: `amount: amount` -> `amount`

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: frontend/src/lib/modals/accounts/IcpTransactionModal.svelte (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/modals/accounts/IcpTransactionModal.svelte]: done

**Pull request merged**